### PR TITLE
added sample app

### PR DESCRIPTION
**Pull request by Harshjosh361**
fix : [#2336](https://github.com/keploy/keploy/issues/2336) 
added sample app for flask-postgresql

### CODE REVIEW

**Comment by Harshjosh361** [comment]
@Swpn0neel can check it once...

**Comment by Harshjosh361** [comment]
still waiting for the review

File: flask_postgresql_app/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Flask==2.0.1
+Flask-SQLAlchemy==2.5.1
+psycopg2-binary==2.9.1
```
Comment (Sonichigo) [inline flask_postgresql_app/requirements.txt:3]: this is an outdated dependency please update it to `2.9.10`

Comment (Sonichigo) [reply flask_postgresql_app/requirements.txt:3]: also since you are using `.env ` please add `pip install python-dotenv`

**Review by Sonichigo — APPROVED** [review]

**Comment by Harshjosh361** [comment]
@Sonichigo i have the changes please check it asap..

**Review by Sonichigo — APPROVED** [review]

**Pull request merged**